Error in rule align_chip:
    jobid: 0
    output: /DATA/projects/DSBrepair/data/xv20220329_HNSCC_H3K27me3_maps/hg38/mapping/HNSCC/Farhangdoost2021/SKN3_GSE149670_r1_SRR11657373.bam.tmp
    log: /DATA/projects/DSBrepair/data/xv20220329_HNSCC_H3K27me3_maps/hg38/log/mapping/HNSCC/Farhangdoost2021/SKN3_GSE149670_r1_SRR11657373.bam.tmp

RuleException:
CalledProcessError in line 796 of /DATA/scratch/usr/c.leemans/projects/chip_snake/chip_align.snakemake:
Command ' set -euo pipefail;  bowtie2 --no-unal -p 10 --met-file /DATA/projects/DSBrepair/data/xv20220329_HNSCC_H3K27me3_maps/hg38/log/mapping/HNSCC/Farhangdoost2021/SKN3_GSE149670_r1_SRR11657373.bam.tmp -x /DATA/data/human/hg38/index/bowtie2/hg38_1-22_XYM -U /DATA/projects/DSBrepair/data/xv20220329_HNSCC_H3K27me3_maps/raw_data/SRR11657373.fastq.gz | sambamba view -S -F "mapping_quality >= 30" -f bam --compression-level=0 /dev/stdin  | sambamba sort -o /DATA/projects/DSBrepair/data/xv20220329_HNSCC_H3K27me3_maps/hg38/mapping/HNSCC/Farhangdoost2021/SKN3_GSE149670_r1_SRR11657373.bam.tmp --compression-level=0 /dev/stdin ' returned non-zero exit status -2
  File "/DATA/scratch/usr/c.leemans/projects/chip_snake/chip_align.snakemake", line 890, in __rule_align_chip
  File "/DATA/scratch/usr/c.leemans/projects/chip_snake/chip_align.snakemake", line 796, in run_align
  File "/home/x.vergara/miniconda3/envs/chip/lib/python3.5/concurrent/futures/thread.py", line 55, in run
